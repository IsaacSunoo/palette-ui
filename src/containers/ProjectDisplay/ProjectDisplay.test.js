import React from 'react';
import { shallow } from 'enzyme'
import SwipeableTemporaryDrawer from './index';

describe('SwipeableTemporaryDrawer', () => {

  describe('SwipeableTemporaryDrawer Component', () => {
    let wrapper;
    let mockSetLock = jest.fn()

    beforeEach(() => {
      wrapper = shallow(<SwipeableTemporaryDrawer/>)
    })

    it('should match the snapshot', () => {
      expect(wrapper).toMatchSnapshot()
    })

  });

});
