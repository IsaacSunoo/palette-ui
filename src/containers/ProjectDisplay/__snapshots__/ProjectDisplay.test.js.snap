// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwipeableTemporaryDrawer SwipeableTemporaryDrawer Component should match the snapshot 1`] = `
<SwipeableTemporaryDrawer
  classes={Object {}}
/>
`;
